src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -129,6 +130,37 @@ public static int parseInsurancePlan(String insurancePlanId) throws ParseExcepti
         return trimmedInsurancePlanId;
     }
 
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given claimId is invalid based on preset conventions.
+     */
+    public static String parseClaimId(String claimId) throws ParseException {
+        requireNonNull(claimId);
+        String trimmedClaimId = claimId.trim();
+        Claim.checkValidClaimId(trimmedClaimId);
+        return trimmedClaimId;
+    }
+
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the claim amount is not a valid positive integer
+     */
+    public static int parseClaimAmount(String claimAmount) throws ParseException {
```

Comment:
Corresponding to comments in #95, I think we should change the handling here to allow for inputs of "30.15" form, without dollar signs, instead of forcing user to enter integer amount. I suggest to use a String method like .replace(".","") to convert "30.15" to "3015" and parse it as int. If anyone has a better suggestion, please comment too!